asserts/model.go

```diff
@@ -246,7 +531,20 @@ func assembleModel(assert assertionBase) (Assertion, error) {
 		return nil, err
 	}
 
-	if classic {
+	// Core 20 extended snaps header
```

Comment:
It would be neat if we just had a `schema` or `version` field and not have to fish for presence of specific model fields.

Reply:
we discussed that and it was nacked